Source/Core/Proj4Projection.js

```diff
@@ -0,0 +1,196 @@
+define([
+        './Cartesian3',
+        './Cartographic',
+        './Math',
+        './Check',
+        './defaultValue',
+        './defined',
+        './defineProperties',
+        './Ellipsoid',
+        './oneTimeWarning',
+        './Rectangle',
+        '../ThirdParty/proj4-2.5.0'
+    ], function(
+        Cartesian3,
+        Cartographic,
+        CesiumMath,
+        Check,
+        defaultValue,
+        defined,
+        defineProperties,
+        Ellipsoid,
+        oneTimeWarning,
+        Rectangle,
+        proj4) {
+    'use strict';
+
+    // proj4 may crash with absolute extents
+    var smallerThanMax = Rectangle.fromDegrees(-180.0 + CesiumMath.EPSILON7, -90.0 + CesiumMath.EPSILON7, 180 - CesiumMath.EPSILON7, 90 - CesiumMath.EPSILON7);
+
+    /**
+     * MapProjection using proj4js. This projection is only to be used with Ellipsoid.WGS84.
+     * Users should exercise caution when using local-area projections, as local area projections
+     * may produce unexpected results outside their valid boundaries in Cartographic space (wgs84Bounds).
+     * These boundaries may be looked up at spatialreference.org.
+     *
+     * Scenes using Proj4Projection will default to <code>MapMode2D.ROTATE</code> instead of <code>MapMode2D.INFINITE_SCROLL</code>.
```

Comment:
> will default to

Is this really a default, or will sing INFINITE not work?  What happens if I specify INFINITE when contsructing the viewer with a WKT projection?

Reply:
It's still settable. `INFINITE_SCROLL` won't crash for projections like polar or mollweide, but it will be very weird and might not make as much sense as rotatable.